src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_ossirian.cpp

```diff
@@ -358,9 +367,28 @@ struct npc_anubisath_guardian : public ScriptedAI
     TaskScheduler _scheduler;
 };
 
+class spell_crystal_weakness : public SpellScript
+{
+    PrepareSpellScript(spell_crystal_weakness);
+
+    void FilterTargets(std::list<WorldObject*>& targets)
+    {
+        targets.remove_if([&](WorldObject const* target) -> bool
+            {
+                return target->GetEntry() != NPC_OSSIRIAN;
```

Comment:
Im not sure about if this will make the spell actually not remove strength from boss?

Reply:
What do you mean? That is already handled on the boss script, this is only to affect Ossirian.